options.html

```diff
@@ -6,7 +6,8 @@
   </head>
 
 <body>
-  <button id="downloadData">Download your data as a CSV file</button>
+  <button id="downloadCategorizationData">Download your categorization data as a CSV file</button>
+  <button id="downloadQaData">Download your QA data as a CSV file</button>
```

Comment:
Here too, use the names suggested above (`Regression/Bug/No-Bug` and `QA Needed`)